gunicorn_thrift/thriftpy_gevent_worker.py

```diff
@@ -1,12 +1,18 @@
 # -*- coding: utf-8 -*-
 
 
+import sys
+import time
 import errno
+import thread
```

Comment:
thread is going to be imported in `logging` anyway, so there's no additional side effects.